docs/maui/views/MediaElement.md

```diff
@@ -343,6 +342,36 @@ To read more about handlers, please see the .NET MAUI documentation on [Handlers
 | Stop | Stops playback and resets the position of the current media. |
 | SeekTo | Takes a `TimeSpan` value to set the `Position` property to and takes a `CancellationToken` to cancel the `Task`. |
 
+## Known issues
+
+> [!WARNING]
+> Possible exception when using full-screen support on Android.
+
+While this is not an issue with the full-screen support in the .NET MAUI Community Toolkit, using the feature will at present, run the risk of exposing a known issue with .NET MAUI - [MAUI Android build crashes when app is reopened from background. It throws the exception: 'Window was already created.'](https://github.com/dotnet/maui/issues/18692).
+
+### Workaround
+
+There is a workaround that can be applied to an applications codebase in order to avoid observing this exception at runtime.
+
+In the *MainActivity.cs* file under */Platforms/Android/*, the following code can be added inside the `MainActivity` class:
+
+```csharp
+protected override void OnPostCreate(Bundle? savedInstanceState)
+{
+    try
+    {
+        base.OnPostCreate(savedInstanceState);
+    }
+    catch(Exception ex)
```

Comment:
Reading through the [linked issue on GitHub](https://github.com/dotnet/maui/issues/18692), it looks like the Exception being thrown is a `InvalidOperationException`.

Let's narrow this catch block down to `InvalidOperationException` to avoid any intended consequences of catching every exception.

Reply:
```suggestion
    catch(InvalidOperationException ex)
```

Reply:
Great spot!